.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+# Pull Request Template
+
+## PR Requirements Checklist
+
+- [ ] The commit message follows our guidelines
+- [ ] Tests for the changes have been added (for bug fixes/features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+## PR Description
```

Comment:
Thoughts on simplifying this to just an SS (if applicable) and something like
```
## Changes in the PR
Provide an itemized list of changes
```
I feel like there is a lot of repeating yourself here

Reply:
To clarify, please keep the requirements section 👍

Reply:
> Thoughts on simplifying this to just an SS (if applicable) and something like
> 
> ```
> ## Changes in the PR
> Provide an itemized list of changes
> ```
> 
> I feel like there is a lot of repeating yourself here

ok, will do